### PR TITLE
Update carousel images, and start on the environmental policy

### PR DESCRIPTION
**Pull request by dcook-net**
Initial stab at the environmental page. Needs a little work in places. Feedback welcome.
Content not viewable to users yet.

I've updated the the images on the carousel to fresher and move diverse.
Logo on carousel is still the old green one, so that will still need changing

### CODE REVIEW

File: DDDEastAnglia/Controllers/HomeController.cs [hunk 1]
```diff
@@ -56,6 +56,11 @@ public ActionResult CodeOfConduct()
         {
             return View();
         }
+        
+//        public ActionResult Environmental()
```
Comment (CraftyFella) [inline DDDEastAnglia/Controllers/HomeController.cs:60]: This correct?

File: DDDEastAnglia/Controllers/NavigationBarController.cs [hunk 2]
```diff
@@ -54,6 +54,7 @@ public ActionResult RenderMenu()
                     CreateLink("Accommodation", "Home", "Accommodation", conference.CanRegister),
                     CreateLink("Sponsors", "Home", "Sponsors"),
                     CreateLink("Team", "Home", "Team"),
+//                    CreateLink("Environmental Policy", "Home", "Environmental"),
```
Comment (CraftyFella) [inline DDDEastAnglia/Controllers/NavigationBarController.cs:57]: And this?

File: DDDEastAnglia/Controllers/HomeController.cs (hunk 1 above)
Comment (dcook-net) [reply DDDEastAnglia/Controllers/HomeController.cs:60]: yeah - still working on it and stupidly started the images on the same fork.
Images are higher priority for now. So keeping the Env stuff dark for now.

Little bit lazy - should have split it all up in to sep change sets!

File: DDDEastAnglia/Controllers/NavigationBarController.cs (hunk 2 above)
Comment (dcook-net) [reply DDDEastAnglia/Controllers/NavigationBarController.cs:57]: as above

File: DDDEastAnglia/Controllers/HomeController.cs (hunk 1 above)
Comment (CraftyFella) [reply DDDEastAnglia/Controllers/HomeController.cs:60]: understood.. just checking it wasn't a mistake

**Pull request merged**